contracts/strategies/StrategyCurveYVoterProxy.sol

```diff
@@ -30,7 +30,7 @@ contract StrategyCurveYVoterProxy {
     address public constant curve = address(0x45F783CCE6B7FF23B2ab2D70e416cdb7D6055f51);
 
     address public constant gauge = address(0xFA712EE4788C042e2B7BB55E6cb8ec569C4530c1);
-    address public constant proxy = address(0x5886E475e163f78CF63d6683AbC7fe8516d12081);
+    address public constant proxy = address(0x5886E475e163f78CF63d6683AbC7fe8516d12081); // TODO Update to fixed StrategyProxy
```

Comment:
Take care of this TODO before merging?

Reply:
Yes, let's wait for the prod greenlight and test to make sure it works before merging.
I'll need to update the exploit test to re-deploy the vulnerable strategy back again after that.

Reply:
@banteg are we merging before deploy? or deploy before merging?